src/network/stream_reader.rs

```diff
@@ -82,11 +81,13 @@ impl<R: BufRead> StreamReader<R> {
 }
 
 #[cfg(test)]
-mod test {
+extern crate tempfile;
```

Comment:
NACK. We just removed this dependency.

Reply:
Was removed yesterday after @elichai pointed out this issue